docs/src/running-validator/restart-cluster.md

```diff
@@ -88,3 +88,28 @@ Post something like the following to #announcements (adjusting the text as appro
 ### Step 7. Wait and listen
 
 Monitor the validators as they restart. Answer questions, help folks,
+
+## Troubleshooting
+
+### 80% of the stake didn't participate in the restart, now what?
+If less than 80% of the stake join the restart after a reasonable amount of
+time, it will be necessary to retry the restart attempt with the stake from the
+non-responsive validators removed.
+
+The community should identity and come to social consensus on the set of
+non-responsive validators. Then all participating validators return to Step 4
+and create a new snapshot with additional `--destake-vote-account <PUBKEY>`
+arguments for each of the non-responsive validator's vote account address
+
+```bash
+$ solana-ledger-tool -l ledger create-snapshot SLOT_X ledger --hard-fork SLOT_X \
+    --destake-vote-account <VOTE_ACCOUNT_1> \
+    --destake-vote-account <VOTE_ACCOUNT_2> \
+    .
+    .
+    --destake-vote-account <VOTE_ACCOUNT_N> \
```

Comment:
🥺 
I'm trying to be optimistic and hope `--destake-vote-account` will never be used.

> I think the interface could be improved though

Anything beyond the config file suggestion though?

Reply:
> :pleading_face:
> I'm trying to be optimistic and hope `--destake-vote-account` will never be used.

testnet, tho

> Anything beyond the config file suggestion though?

nope! lgtm otherwise

Reply:
Do you have strong objections to leaving `--destake-vote-account` as is for now? I feel like a wrapper shell script could be easily created to deal with copy-paste issues, and exceeding `ARG_MAX` vote accounts to destake is just too sad of a thought to consider.

Reply:
Fair enough.  If we get to the point where default `ARG_MAX` is a problem, we'll definitely have restarted the cluster a time or two 😅